src/Core/Feature.js

```diff
@@ -332,4 +332,18 @@ export class FeatureCollection {
         this.features.push(ref);
         return ref;
     }
+
+    /**
+     * Transforms a given {@link Coordinates}, using the translation and the
+     * scale of this collection.
+     *
+     * @param {Coordinates} coordinates - The coordinates to transform
+     *
+     * @return {Coordinates} The same coordinates, with transformation applied.
+     */
+    transformCoordinates(coordinates) {
+        coordinates.x = (coordinates.x / this.scale.x) - this.translation.x;
```

Comment:
Could you add z/altitude transformation ?

Reply:
Done, I thought they were `Vector2`